ts/src/binance.ts

```diff
@@ -1952,6 +1952,8 @@ export default class binance extends Exchange {
                         '-4088': PermissionDenied, // User can not place order currently
                         '-4114': BadRequest, // INVALID_CLIENT_TRAN_ID_LEN
                         '-4115': BadRequest, // DUPLICATED_CLIENT_TRAN_ID
+                        '-4116': BadRequest, // DUPLICATED_CLIENT_ORDER_ID
```

Comment:
@ttodua shouldn't this be a `InvalidOrder`?

Reply:
could be, yeah. pushed